include/veriblock/blockchain/block_index.hpp

```diff
@@ -48,6 +48,23 @@ struct BlockIndex {
     return pbegin[(pend - pbegin) / 2];
   }
 
+  const BlockIndex* getAncestorBlocksBehind(height_t steps) const {
+    if (steps < 0 || steps > this->height) {
+      return nullptr;
+    }
+
+    if (steps == 0) {
```

Comment:
Checks for steps are not necessary. Code will work the same way with an assumption that steps <= 0 return current block.

Reply:
fixed